src/linttest/linttest.go

```diff
@@ -79,7 +82,7 @@ type Suite struct {
 	AllowDisable *regexp.Regexp
 
 	defaultStubs map[string]struct{}
-	LoadStubs    []string
+	LoadedStubs  []string
```

Comment:
It's not obvious why this renaming is needed.
I don't know why we need a setter method for it as well.

Reply:
Deleted